src/registrar/config/settings.py

```diff
@@ -251,8 +251,7 @@
 BOTO_CONFIG = Config(retries={"mode": AWS_RETRY_MODE, "max_attempts": AWS_MAX_ATTEMPTS})
 
 # email address to use for various automated correspondence
-# TODO: pick something sensible here
-DEFAULT_FROM_EMAIL = "registrar@get.gov"
+DEFAULT_FROM_EMAIL = "help@get.gov"
```

Comment:
```suggestion
DEFAULT_FROM_EMAIL = "help@get.gov <help@get.gov>"
```

Reply:
This will set the "Friendly from", which will look better in list view on some (all?) email clients:

![Screen Shot 2023-08-28 at 5 16 27 PM](https://github.com/cisagov/getgov/assets/107004823/b165869d-68d2-48b7-9079-20bf31322a51)